### PR TITLE
just test

### PR DESCRIPTION
**Pull request by KaterynaLutska**
add new title

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 
 function App() {
   return (<>
-    <h1>LETS GET STARTED!</h1>
+    <h1>LETS MAKE PROJECT!</h1>
```
Comment (Huracan22505) [inline src/App.js:4]: You need use text-transform in CSS

try

`h1 {
  text-transform: uppercase;
}`

**Review by Huracan22505 — CHANGES_REQUESTED** [review]
Hi! Good work, but you have 1 mistake.

File: src/App.js (hunk 1 above)
Comment (Huracan22505) [reply src/App.js:4]: Allow only one h1 tag on page

Comment (KaterynaLutska) [reply src/App.js:4]: It wasn`t written in TT)

Comment (KaterynaLutska) [reply src/App.js:4]: but i will add it

Comment (KaterynaLutska) [reply src/App.js:4]: fixed

**Review by Huracan22505 — APPROVED** [review]

**Pull request closed without merging**